src/StorybookStyles.js

```diff
@@ -0,0 +1,17 @@
+// import React from 'react'
```

Comment:
Delete comments instead of pushing them to master, if the file is empty delete it.